conda-recipe/meta.yaml

```diff
@@ -24,6 +24,8 @@ requirements:
     - filestore
     - matplotlib
     - prettytable
+    - cycler
+    - traitlets
```

Comment:
I only ask because I think the plan is to change `.condarc` on each beamline so that they point to `anaconda`, which hits the internal server only, not `defaults`, which hits the public server too.  If we don't have these two deps on our internal server then the install will fail (yeah i guess that's obvious, sorry for the stupid comment)